core/src/main/java/io/parsingdata/metal/data/ParseValueList.java

```diff
@@ -55,18 +55,18 @@ public ParseValueList add(final ParseValueList list) {
 
     public ParseValue get(final String name) {
         if (isEmpty()) { return null; }
-        if (head.matches(name)) {
-            return head;
-        } else {
+        if (head.matches(name)) { return head; }
+        else {
             return tail.get(name);
         }
     }
 
     public ParseValueList getAll(final String name) {
         if (isEmpty()) { return this; }
         final ParseValueList tailList = tail.getAll(name);
-        if (head.matches(name)) { return tailList.add(head); }
-        else { return tailList; }
+        if (head.matches(name)) {
+            return tailList.add(head);
+        } else { return tailList; }
```

Comment:
This looks so counterintuitive, to have a multi-line `if` and a single line `else`. But I guess it does follow our rule 😉